src/stylesheets/components/_results.scss

```diff
@@ -9,4 +9,29 @@
     justify-content: center;
     flex-direction: column;
   }
-}
+
+  &.dual-record {
+    background-color: #f9f9f9;
```

Comment:
we need to use one of our base colors rather than hard code a hex

Reply:
Use #f9f9f7

Reply:
changed to #f9f9f7 (per Kapil)